Makefile

```diff
@@ -13,7 +16,13 @@ test-clj: .source-deps
 	lein with-profile +$(VERSION),+test-clj test
 
 test-cljs: .source-deps
-	lein with-profile +$(VERSION),+test-cljs test
+	if [ "$(JAVA_VERSION)" = "9.0" ] || [ "$(JAVA_VERSION)" = "9" ]; then \
+            lein with-profile +$(VERSION),+test-cljs \
+                 update-in :jvm-opts concat '["--add-modules" "java.xml.bindgn"]' \
```

Comment:
Thanks a lot @gonewest818! I've pretty much used your code verbatim, except for using single quotes here.  Escaping the double quotes didn't work.

The `jxb` profile sure was ugly. This looks so much better.